eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -74,401 +87,508 @@ Ref to search for
+    $uri = "$GithubAPIBaseURI/$RepoOwner/$RepoName/issues/$IssueNumber/comments"
+
+    $parameters = @{
+        body = $Comment
+    }
+
+    return Invoke-RestMethod `
+        -Method POST `
+        -Body ($parameters | ConvertTo-Json) `
+        -Uri $uri `
+        -Headers (Get-GitHubApiHeaders -token $AuthToken) `
+        -MaximumRetryCount 3
 }
 
 # Will add labels to existing labels on the issue
 function Add-GitHubIssueLabels {
-  param (
-    [Parameter(Mandatory = $true)]
-    $RepoOwner,
-    [Parameter(Mandatory = $true)]
-    $RepoName,
-    [Parameter(Mandatory = $true)]
-    $IssueNumber,
-    [ValidateNotNullOrEmpty()]
-    [Parameter(Mandatory = $true)]
-    $Labels,
-    [ValidateNotNullOrEmpty()]
-    [Parameter(Mandatory = $true)]
-    $AuthToken
-  )
-
-  if ($Labels.Trim().Length -eq 0)
-  {
-    throw " The 'Labels' parameter should not not be whitespace.
+    param (
+        [Parameter(Mandatory = $true)]
+        $RepoOwner,
+        [Parameter(Mandatory = $true)]
+        $RepoName,
+        [Parameter(Mandatory = $true)]
+        $IssueNumber,
+        [ValidateNotNullOrEmpty()]
+        [Parameter(Mandatory = $true)]
+        $Labels,
+        [ValidateNotNullOrEmpty()]
+        [Parameter(Mandatory = $true)]
+        $AuthToken
+    )
+
+    if ($Labels.Trim().Length -eq 0) {
+        throw " The 'Labels' parameter should not not be whitespace.
     Use the 'Update-Issue' function if you plan to reset the labels"
-  }
-
-  $uri = "$GithubAPIBaseURI/$RepoOwner/$RepoName/issues/$IssueNumber/labels"
-  $parameters = @{}
-  $parameters = Set-GitHubAPIParameters -members $Labels -parameterName "labels" `
-  -parameters $parameters
-
-  return Invoke-RestMethod `
-          -Method POST `
-          -Body ($parameters | ConvertTo-Json) `
-          -Uri $uri `
-          -Headers (Get-GitHubApiHeaders -token $AuthToken) `
-          -MaximumRetryCount 3
+    }
+
+    $uri = "$GithubAPIBaseURI/$RepoOwner/$RepoName/issues/$IssueNumber/labels"
+    $parameters = @{}
+    $parameters = Set-GitHubAPIParameters -members $Labels -parameterName "labels" `
+        -parameters $parameters
+
+    return Invoke-RestMethod `
+        -Method POST `
+        -Body ($parameters | ConvertTo-Json) `
+        -Uri $uri `
+        -Headers (Get-GitHubApiHeaders -token $AuthToken) `
+        -MaximumRetryCount 3
+}
+function Get-GitHubIssueLabels {
+    param (
+        [Parameter(Mandatory = $true)]
+        $RepoOwner,
+        [Parameter(Mandatory = $true)]
+        $RepoName,
+        [Parameter(Mandatory = $true)]
+        $IssueNumber,
+        [ValidateNotNullOrEmpty()]
+        [Parameter(Mandatory = $true)]
+        $AuthToken
+    )
+  
+    $uri = "$GithubAPIBaseURI/$RepoOwner/$RepoName/issues/$IssueNumber/labels"
+  
+    return Invoke-RestMethod `
+        -Method GET `
+        -Uri $uri `
+        -Headers (Get-GitHubApiHeaders -token $AuthToken) `
+        -MaximumRetryCount 3
+}
+function Remove-GitHubIssueLabels {
```

Comment:
Let's hold off adding this functions as we are planning to use a common GitHub action to process label addition and deletion. CC: @mikeharder